### PR TITLE
Add mention of the SymfonyTestsListener

### PR DESCRIPTION
**Pull request by theofidry**
Follow up of https://github.com/symfony/symfony/issues/21982#event-997000232.

/cc @xabbuh

### CODE REVIEW

**Review by javiereguiluz — APPROVED** [review]
👍 

@theofidry thanks for completing the docs about this feature. I did some rewords, but I maintained most of your original contribution. Thanks!

File: components/phpunit_bridge.rst [hunk 1]
```diff
@@ -35,6 +35,24 @@ You can install the component in 2 different ways:
 
 .. include:: /components/require_autoload.rst.inc
 
+If you plan to :ref:`write-assertions-about-deprecations`_ and use the regular
```
Comment (xabbuh) [inline components/phpunit_bridge.rst]: the trailing underscore must be removed

File: components/phpunit_bridge.rst [hunk 2]
```diff
@@ -35,6 +35,24 @@ You can install the component in 2 different ways:
...
+PHPUnit script (not the modified PHPUnit script provided by Symfony), then you
+must register a new `test listener`_ called ``SymfonyTestsListener``:
```
Comment (xabbuh) [inline components/phpunit_bridge.rst]: [...] then you must register [...] -> [...] you have to register [...]?

**Review by HeahDude — APPROVED** [review]

**Comment by theofidry** [comment]
Thanks very much :)

**Comment by xabbuh** [comment]
Thank you @theofidry.

**Pull request closed without merging**